plugins/tactics/src/Ide/Plugin/Tactic.hs

```diff
@@ -110,11 +114,12 @@ commandProvider HomomorphismLambdaCase =
 
 ------------------------------------------------------------------------------
 -- | A mapping from tactic commands to actual tactics for refinery.
-commandTactic :: TacticCommand -> OccName -> TacticsM ()
+commandTactic :: TacticCommand -> String -> TacticsM ()
```

Comment:
Why this change?

Reply:
`OccName` is a string + a namespace. Data constructors live in a different namespace from variables, which is the reason that https://github.com/isovector/haskell-language-server/pull/30/files#diff-98860ec806a30382db306b93440066959710bb5663b2ac0fef2201e27451797fR120-R122 uses `mkDataOcc` and `mkVarOcc` respectively and https://github.com/isovector/haskell-language-server/pull/30/files#diff-98860ec806a30382db306b93440066959710bb5663b2ac0fef2201e27451797fR120-R122 can't just always use `mkVarOcc`

Reply:
Excellent catch. I'd missed that entirely, thanks!